docs/admin/install.rst

```diff
@@ -305,10 +311,11 @@ With the key and configuration available in ``dom0``, you're ready to set up Sec
   .. code-block:: none
 
     securedrop-workstation-dom0-config-<versionNumber>-1.fc25.noarch.rpm:
-      Header V4 RSA/SHA256 Signature, key ID 7b22e6a3: OK
+      Header V4 RSA/SHA512 Signature, key ID 7b22e6a3: OK
+      Header SHA256 digest: OK
       Header SHA1 digest: OK
-      V4 RSA/SHA256 Signature, key ID 7b22e6a3: OK
-      MD5 digest: OK
+      Payload SHA256 digest: OK
+      V4 RSA/SHA512 Signature, key ID 7b22e6a3: OK
```

Comment:
I also see `MD5 digest: OK` in my output - if that's consistently part of the output I think we should include it here to match the full output.

Reply:
Good catch, @eloquence.  Fixed in 441cc36bccdd681e7382e5d53bafab0f512286e0.